src/swift/codeGeneration.js

```diff
@@ -376,6 +381,15 @@ export function structDeclarationForSelectionSet(
   });
 }
 
+function operationIdentifier(generator,  { operationName, sourceWithFragments, operationId }) {
+  if (!generator.context.generateOperationIds) {
+    return
+  }
+
+  generator.printNewlineIfNeeded();
+  generator.printOnNewline(`public static let operationId = "${operationId}"`);
```

Comment:
This is definitely a bit nitpicky, but would `operationID` or even `operationIdentifier` be more Swifty?